orttraining/orttraining/core/optimizer/megatron_transformer.cc

```diff
@@ -291,30 +292,33 @@ Status MegatronTransformer::TransformMLP(Graph& graph, bool& modified, int graph
 
     auto a_weight_arg = node.MutableInputDefs()[1];
     ONNX_NAMESPACE::TensorProto a_weight_initializer_partition;
-    if (!PartitionWeightByColumn(graph, *a_weight_arg, a_weight_initializer_partition)) {
```

Comment:
seems more of our code perfer to have such spaces between if and (

Reply:
have run the formatter